doc/source/webapp_ui/preview/graph.rst

```diff
@@ -0,0 +1,10 @@
+======================
+Workflow Preview Graph
+======================
+
+The Workflow Preview Graph shows the transport graph of the lattice on which the :code:`lattice.draw()` function was last called.
```

Comment:
should we also mention the use case? Often times this is useful to "view" the costly experiment that would run before it actually uses the resources.

Reply:
Done.